app/i18n/wallet.en.i18n.json

```diff
@@ -304,6 +304,8 @@
             "buttons": {
                 "showContractInfo": "Show contract info",
                 "hideContractInfo": "Hide contract info",
+                "watchContractEvents": "Watch contract events",
+                "stopWatchingContractEvents": "Stop watching contract events",
```

Comment:
I think a better action would be "Load Latest Events" or simply "Load Events"/"Stop loading events". Since the button states does not persists after you leave the page, and having a wording that seems to indicate this is a state change is misleading. Of course, the app won't stop loading the events while you are there, but then there is also a new block incoming so it would make sense for it not to.